### DISCUSSION TITLE
Elsa 2.4.0.1 - Workflow starts in Idle status

### DISCUSSION THREAD

**Discussion by stanroxana** [elsa-workflows/elsa-core#2677] [Q&A]
Hello, I am trying to start a workflow (simple, for testing purposes only) on a machine that runs 2 instances of my application, both with Elsa server running. Any workflow that I try to start has the status _Idle_.
On my local machine the execution works correctly, but with only one instance running. 

Is there any known limitation regarding running Elsa on multiple instances on the same machine?

Thank you for your time and for this great library! :)

**Comment by sfmskywalker** [comment]
There aren't any limitations when running multiple instances of Elsa on the same or even multiple machines.

You will have to configure your setup to allow this distributed model as described [here](https://elsa-workflows.github.io/elsa-core/docs/hosting/hosting-distributed-hosting).

Let me know if you have any questions about that!
